include/pipettes/core/messages.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <functional>
+
+#include "common/core/i2c.hpp"
+
+namespace pipette_messages {
+
+using MaxBufferSize = std::array<uint8_t, 5>;
+using Callback = std::function<void(MaxBufferSize)>;
```

Comment:
```suggestion
using Callback = std::function<void(const MaxBufferSize&)>;
```
though it doesn't really matter at 5 bytes